src/utils/language.ts

```
@@ -0,0 +1,9 @@
+function getUserLanguage(): string {
+  if (navigator.language && navigator.language.startsWith('es')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)